src/Okta.Sdk/OktaClient.cs

```diff
@@ -144,6 +146,9 @@ private static OktaClientConfiguration GetConfigurationOrDefault(OktaClientConfi
             return compiledConfig;
         }
 
+        /// <inheritdoc/>
+        public RequestContext RequestContext => _requestContext;
```

Comment:
As discussed, let's try to avoid exposing this or ultimately making sure is read-only.